assets/js/components/setup/site-verification.js

```diff
@@ -68,11 +71,17 @@ class SiteVerification extends Component {
 
 				// Our current siteURL has been verified. Proceed to next step.
 				if ( verified ) {
+					sendAnalyticsTrackingEvent( 'verification_setup', 'verification_check_true' );
+
 					const response = await this.insertSiteVerification( identifier );
 					if ( true === response.updated ) {
+						sendAnalyticsTrackingEvent( 'verification_setup', 'verification_insert_tag' );
```

Comment:
This shouldn't be right here: The user is already verified at this point, so all the request does is set the flag on the server that the user is verified. It should instead follow the `insertSiteVerification()` call further down in the file, which is called when the user wasn't verified before (in this case it will actually insert the tag, which is what this tracking event is for).